WMDE_FR_2025_Desktop_DE_11

moiikana opened the pull request:
- based on WMDE_FR_2025_Desktop_DE_10 VAR
- VAR will have new mustard theme/design

https://phabricator.wikimedia.org/T410120

moiikana committed:
Prepare campaign for WMDE_FR_2025_Desktop_DE_11

- based on WMDE_FR_2025_Desktop_DE_10 VAR
- VAR will have new mustard theme/design

https://phabricator.wikimedia.org/T410120

moiikana committed:
Prepare CTRL for WMDE_FR_2025_Desktop_DE_11

- based on WMDE_FR_2025_Desktop_DE_10 VAR

https://phabricator.wikimedia.org/T410120

moiikana committed:
Implement VAR for WMDE_FR_2025_Desktop_DE_11

- new design (orange, beige)
- new use of funds sub component in theme
- changes sizes of right column, close button, moves position of left column

https://phabricator.wikimedia.org/T410120

moiikana committed:
Move start date

moiikana committed:
Implement overlapping layout ( WMDE_FR_2025_Desktop_DE_11 )

- this creates an overlapped style for the message box and the forms (main donation form
and upsell form)
- makes submit button overlap the lower border

- implement darkmode colors

- fix unit tests for changed footer

https://phabricator.wikimedia.org/T410120

Abban commented on banners/desktop/WMDE_FR_2025_Desktop_DE_11/styles/MainBanner_var.scss:
```diff
@@ -0,0 +1,70 @@
+		flex-direction: column;
+		width: $form-width;
+		padding: 0;
+		background: linear-gradient(
```
This is a lovely way to handle the bg overlap

Abban reviewed (commented):
When you get to the second page of the form the first page is `opacity: 0` but it's possible to still interact with it

![Image](https://github.com/user-attachments/assets/69749f08-3bdf-4176-ae2c-5cc82663d794)

Abban commented:
Here's a fix for the issue https://github.com/wmde/fundraising-banners/pull/883

Abban committed:
Fix hidden form being interactable (#883)

The new banner CSS made the main donation form
hidden but it was still interactable. This updates
some styles to make sure it has the display removed.

Also fixes linting errors.

Ticket: https://phabricator.wikimedia.org/T410120
